src/index.js

```diff
@@ -10,4 +11,21 @@ const server = app.listen(port, () => {
   auditLogger.info(`Listening on port ${port}`);
 });
 
```

Comment:
We could obviously put some logic in this parsed message stuff, I think the approach I'd probably take is to maintain the connections/clients in little arrays somehow, based on report id

Reply:
We can't store any state info in memory since prod has multiple instances. Will have to be in postgres or redis (or something else that we spin up just for this purpose)

Reply:
I'm looking at the redis pub/sub stuff right now, seems like a pretty decent solution and we shouldn't have to add anything infrastructure-wise

Reply:
ex: https://gist.github.com/pietern/348262